controllers/dscinitialization/dscinitialization_controller.go

```diff
@@ -212,11 +214,23 @@ func (r *DSCInitializationReconciler) Reconcile(ctx context.Context, req ctrl.Re
 
 		return ctrl.Result{}, nil
 	default:
+		createUsergroup, err := cluster.IsDefaultAuthMethod(ctx, r.Client)
+		if err != nil {
+			if !k8serr.IsNotFound(err) { // only keep reconcile if real error but not missing CRD or missing CR
```

Comment:
nit: it can probably be a single line `if err != nil && !k8serr.IsNotFound(err)`